config/routes.rb

```diff
@@ -1,11 +1,12 @@
 Rails.application.routes.draw do
   scope"(:locale)", locale: /en|vi/ do
-  root "application#hello"
+  root "static_pages#home"
 
   get "/home", to: "static_pages#home"
   get "/help", to: "static_pages#help"
   get "/contact", to: "static_pages#contact"
   get "/about", to: "static_pages#about"
-  get "/new", to: "users#new"
+  get "/signup", to: "users#new"
+  resources :users
```

Comment:
- lùi vào 2 spaces cho toàn bộ dòng này
- trước resources cách 1 dòng trắng ra em